Sources/Fingertips/FingerTipWindow.swift

```diff
@@ -200,13 +202,6 @@ public class FingerTipWindow: UIWindow {
         perform(#selector(removeInactiveFingerTips), with: nil, afterDelay: 0.1)
     }
 
-    func cancelScheduledFingerTipRemoval() {
-        fingerTipRemovalScheduled = true
-        NSObject.cancelPreviousPerformRequests(withTarget: self,
-                                               selector: #selector(removeInactiveFingerTips),
-                                               object: nil)
-    }
-
```

Comment:
This is not used; I imagine that it was once used in conjunction with the `perform(#selector(removeInactiveFingerTips), with: nil, afterDelay: 0.1)`. I do not see a need for it, but perhaps I'm not seeing the full picture.